src/mam/mod_mam_muc_rdbms_arch.erl

```diff
@@ -127,12 +127,13 @@ archive_size(Size, Host, RoomID, _RoomJID) when is_integer(Size) ->
 -spec archive_message(_Result, jid:server(), MessID :: mod_mam:message_id(),
                       RoomID :: mod_mam:archive_id(), _LocJID :: jid:jid(),
                       SenderJID :: jid:jid(),
-                      UserRoomJID :: jid:jid(), incoming, Packet :: packet()) -> ok.
+                      UserRoomJID :: jid:jid(), none, incoming, Packet :: packet()) -> ok.
```

Comment:
Ag, I guess I missed this one, sorry. Spec should be again `binary() | none`. Maybe we could improve indentation already, `SenderJID` has a line for himself and then the next 4 parameters and the return value are all on the next.